packages/nimble-components/src/table-column/number-text/index.ts

```diff
@@ -8,14 +8,15 @@ import { TableColumnSortOperation } from '../base/types';
 import { tableColumnNumberTextGroupHeaderTag } from './group-header-view';
 import { tableColumnNumberTextCellViewTag } from './cell-view';
 import type { ColumnInternalsOptions } from '../base/models/column-internals';
-import { NumberTextFormat } from './types';
+import { NumberTextAlignment, NumberTextFormat } from './types';
 import type { NumberFormatter } from './models/number-formatter';
 import { RoundToIntegerFormatter } from './models/round-to-integer-formatter';
 import { DefaultFormatter } from './models/default-formatter';
 
 export type TableColumnNumberTextCellRecord = TableNumberField<'value'>;
 export interface TableColumnNumberTextColumnConfig {
     formatter: NumberFormatter;
+    rightAlign: boolean;
```

Comment:
Is there any situation that we would center-align? I guess if one comes up, we can change this to `alignment` at that point.

Reply:
I'll leave this conversation open so that others can weigh in if they have an opinion. I think that it makes sense for the property in `TableColumnNumberTextColumnConfig` to stay as `rightAlign`, but I think the bigger question is whether or not we want the alignment in `TableColumnTextCellViewBase` to be more flexible (i.e. an enum of `right`, `left`, `center` rather than a `boolean` indicating right vs left). The `TableColumnTextCellViewBase` will have a larger impact if we decide to change it in the future than internal config for this specific column.

Reply:
I agree that number columns should only support right and left. I bet we'll never support center alignment there.

I could imagine other text columns wanting center alignment someday. So if it's simple to add to the base class now but might be complex to do later, then I'd lean towards adding it now.

Reply:
My biggest concern about adding the enum was that I didn't want to add a mode that we didn't actually need & didn't have a way to have UI tests for. I'm not sure why I didn't think about it until now, but I'll go ahead and change the boolean to an enum but only support "right" and "left" for now. Then, we can wait to add support for "center" until we need it, but it won't be a breaking change that time.